rllib/core/rl_module/torch/torch_rl_module.py

```diff
@@ -29,20 +29,15 @@ def get_state(self) -> Mapping[str, Any]:
     def set_state(self, state_dict: Mapping[str, Any]) -> None:
         self.load_state_dict(state_dict)
 
+    def _weights_relative_path(self) -> pathlib.Path:
+        return pathlib.Path("module_state.pt")
```

Comment:
And this one is no directory and just module_state.pt? I see an inconsistency between tf and torch in terms of the hierarchy of the file structure in which the final files will sit on.

Reply:
fixed